CONTRIBUTING_TR.md

```diff
@@ -0,0 +1,12 @@
+HaxeFlixel'e katkıda bulunduğunuz için teşekkürler!
```

Comment:
this file should should be in the .github folder with the original CONTRIBUTING.md